spec/features/discounts/show_spec.rb

```diff
@@ -30,7 +30,7 @@
       fill_in 'Email', with: user.email
       fill_in 'Password', with: user.password
       click_button 'Log In'
-      visit merchant_discount_path(discount1.id)
+      visit "/merchant/discounts/#{discount1.id}"
```

Comment:
I had to switch this to specify the URI because I lost the path helper when I removed the namespace routes.